imagehash.py

```diff
@@ -495,6 +506,8 @@ def _find_region(remaining_pixels, segmented_pixels):
 def _find_all_segments(pixels, segment_threshold, min_segment_size):
 	"""
 	Finds all the regions within an image pixel array, and returns a list of the regions.
+	WARNING: You will get slightly different segmentation between pillow version 6 and version 7, due to a rounding
```

Comment:
Add a newline before the warning, and change to:
`Note: Slightly different segmentations are produced when using pillow version 6 vs. >=7, due to a change in rounding in the greyscale conversion.`

Reply:
Sounds good, swapped to that wording in 2b9f2e1e6b5d7e891b0c139e8de923452b29908e